src/logrotate.py

```diff
@@ -134,7 +137,7 @@ def _write_config(logrotate_config: LogrotateConfig) -> None:
         f"""{logrotate_config.log_path_glob_pattern} {{
 {logrotate_config.frequency}
 rotate {logrotate_config.rotate}
-copytruncate
+{"copytruncate" if logrotate_config.copytruncate else "nocopytruncate"}
```

Comment:
Would it be helpful to leave a comment on why copytruncate was used in the first place? (I don't quite remember why, I'm thinking whether we should remove `copytruncate` completely)

Reply:
I think @yhaliaw added it , probably because the external application doesn't implement a HUP to reload on a logrotate and would write to a deleted file. I also think we should move to WatchedFileHandlers,but for reactive processes this needs redesign and I don't want to tackle this in this PR.